src/main/webapp/js/utils/filter/text-filter.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Input from '../inputs/input';
+
+const TextFilter = ({ onChange }) => (
+  <Input onChange={onChange} />
```

Comment:
I think it will be best to use it as controlled component, pass a value to the Input

Reply:
Ok. Why text-filter shouldbe a controlled component and enum-filter not?

Reply:
enum filter is controlled, because you're passing the value of the filter from the properties to the select